pages/team.md

```diff
@@ -46,7 +46,7 @@ Kay is a technical writer at Inrupt and a newcomer to Solid.
 
 ## Kyra Assaad
 **Creator**
-Kyra Assaad is the Product Designer at Inrupt. She works to ensure that the Solid user experience is accessible and useable for all. She serves as one of the chairs of (and helped form) the Solid Diversity, Equity, and Inclusion (DEI) team. Whose goal is to increase voices in the Solid community by inviting individuals from all underrepresented groups..
+Kyra Assaad is the Product Designer at Inrupt. She works to ensure that the Solid user experience is accessible and useable for all. She helped form and now serves as one of the chairs of the Solid Diversity, Equity, and Inclusion (DEI) team. She is passionate about increasing representation and participation of women and people of color in the Solid community.
```

Comment:
That works for me,  but from your other comment, do you want add the tie to your motivation?

```suggestion
Kyra Assaad is the Product Designer at Inrupt. She works to ensure that the Solid user experience is accessible and useable for all. Passionate about increasing representation and participation of women and people of color in the Solid community, she has helped form and now serves as one of the chairs of the Solid Diversity, Equity, and Inclusion (DEI) team. 
```

Reply:
heh, I commented before your last commit.

Reply:
That's the one. Thank you so much for helping me!